tasks/Game of Clue - Terrorist Attack.md

```diff
@@ -26,7 +26,35 @@ See supplemental materials PDF for images of computer interface
 
 
 ## Instructions
-Reached out to researchers 
+Instructions on computer screen created by CSS Lab: 
+Thank you for participating in our study, please watch the following video about the task you’ll be working on today. You will be participating an a task involving piecing together clues to figure out the components of a terrorist attack. Specifically, 
+-	Who would carry out the attack (group involved))
+-	What would be the target (e.g., an embassy or a church)
+-	Where the attack would take place (country) 
+-	When the attack would take place (with four interdependent components—month, day, hour, and time of day (a.m. versus p.m.))
+
+Participants watch video and are returned to their inbox (below are descriptions of how the tasks work that could be included in a video or presented as-is)
+![image](https://user-images.githubusercontent.com/78745728/126653865-a7a7726c-b60b-40d2-9ddb-33cc52a9e086.png)
```

Comment:
Adding a line above (and between?) images makes them layout a little better in markdown. (otherwise it sometimes tries to put them inline with the document)

```suggestion

![image](https://user-images.githubusercontent.com/78745728/126653865-a7a7726c-b60b-40d2-9ddb-33cc52a9e086.png)
```